recipes-kernel/kernel-tests/kernel-performance-tests-files/run-cyclictest

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh
+source ./common.cfg
+
+function add_system_info()
+{
+    {
+	DESC=$(fw_printenv DeviceDesc)
```

Comment:
Could you add a dependency on the `fw-printenv` package, which apparently [provides the utility](https://github.com/ni/meta-nilrt/blob/nilrt/master/sumo/recipes-ni/ni-utils/fw-printenv_1.0.bb).

Reply:
Sounds good. Thanks for catching this.